.env

```diff
@@ -91,14 +91,21 @@ ENVOY_IMAGE_TAG=v1.22.0
 MIRROR_NODE_EXPLORER_IMAGE_PREFIX=gcr.io/hedera-registry/
 MIRROR_NODE_EXPLORER_IMAGE_TAG=24.3.0
 
-### Mirror Node Explorer ###
-DOCKER_LOCAL_MIRROR_NODE_MENU_NAME=LOCALNET
-DOCKER_LOCAL_MIRROR_NODE_URL=http://127.0.0.1:5551/
-
 ### Prometheus ####
 PROMETHEUS_IMAGE_NAME=prom/prometheus
 PROMETHEUS_IMAGE_TAG=v2.41.0
 
 ### Grafana ####
 GRAFANA_IMAGE_NAME=grafana/grafana
 GRAFANA_IMAGE_TAG=8.5.16
+
+### Sourcify ####
+SOURCIFY_TAG=main
+
+#Sourcify Server
+SOURCIFY_SERVER_PORT=5555
+SOURCIFY_NODE_ENV=development
+
+#Sourcify ui
+SOURCIFY_UI_PORT=1234
```

Comment:
Is it possible to use the same external port that `hedera-sourcify` uses for `ui`, _i.e._, `3001`?

https://github.com/hashgraph/hedera-sourcify/blob/e313ed01504026ad6e0884b786da2a20c3305593/.env#L7

It's better if we provide a consistent DX across repos.